### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by rogerkuou**
- [x] Intro
- [x] Usage
- [x] Example notebook
- [x] Update zenodo example dataset

### CODE REVIEW

**Comment by rogerkuou** [comment]
Hi @fnattino, could you please review the docs for STM? It is also deployed at: https://motionbylearning.github.io/stmtools/ .

Thanks!

File: docs/operations.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+When `soil_map.gpkg` is small, loading it as an `GeoDataFrame` is faster:
+
+```python
+import geopandas as gpd
+polygon = gpd.read_file('soil_map.gpkg')
+fields_to_query = ['soil_type', 'type_code']
+stmat_enriched = stmat.stm.enrich_from_polygon(path_polygon, fields_to_query)
+```
+
+## Subset an STM
```
Comment (fnattino) [inline docs/operations.md:28]: Just out of curiosity, how are these different from the standard Xarray filtering? If there are any advantages of doing these tasks via STMTools, maybe worth mentioning it?

**Review by fnattino — COMMENTED** [review]
Hey @rogerkuou, again, nice docs (and beautiful logo!)! 

Just some minor stuff that you might (not) want to address. For my side feel free to merge when you deem it's complete!

File: docs/operations.md (hunk 1 above)
Comment (rogerkuou) [reply docs/operations.md:28]: Indeed it's actually the same. We make it an option in "subset" to suit an existing behavior in their current MATLAB implementation.

I added the description of the equivalent filtering in Xarray. What we can do in the future is not forcing the evaluation of the mask. Documented this in issue #50

**Pull request merged**